AEM/mlperf/README.md

```diff
@@ -7,9 +7,720 @@ translation, and others. The benchmark is representative of real-world
 workloads and as a fair and useful way to compare the performance of different
 machine learning systems.
 
+In this document, we'll show how to run Intel MLPerf v3.1 submission with Intel
+optimized Docker images.
+
+## Intel Docker Images for MLPerf
+
+The Intel optimized Docker images for MLPerf v3.1 can be built using the Dockerfiles.
+Example for building docker image with Dockerfile:
+```
+cd <THIS_REPO>/closed/Intel/code/resnet50/pytorch-cpu/docker/
```

Comment:
you need to provide github link directly. it is an IDZ article instead of a markdown file in github repo.